src/components/fileviewer.js

```diff
@@ -201,19 +215,26 @@ const Line = ({ lineNumber, lineText, coverage, selectedLine, onLineClick }) =>
 };
 
 /* This component contains metadata of the file */
-const FileViewerMeta = ({ revision, path, appError, fetchStatus }) => {
+const FileViewerMeta = ({ revision, path, appError, srcFetchStatus, covFetchstatus }) => {
   const showStatus = (label, status) => {
-    const msg = status ? <span>&#x2714;</span> : 'Fetching...';
+    let msg = 'Fetching...';
+    if (status === true) {
```

Comment:
Please fix this if/else block so the trinary state is obvious. By checking for the `undefined` option you simplify the truthiness checks too.:

```javascript
if (status===undefined){
    msg = 'Fetching...';
}else if (status){
    msg = <span>&#x2714;</span>; 
}else{
     msg = <span>&#x2716;</span>; 
}
```